### DISCUSSION TITLE
Would be nice to infer Ulids as strings

### DISCUSSION THREAD

**Discussion by maukoese** [dedoc/scramble#220] [Ideas]
When using ULID instead of UUID, the path parameter with route model binding (or even resources), is being inferred as an integer. Would be nice to make it a string.

I understand ULIDs are newer, but if UUIDs are supported this shouldn't be much of an ask, right?

This is an amazing job you guys are doing

**Comment by romalytvynenko** [comment]
@maukoese 

It would be great, but how can I represent it in Open API spec? 

If you can help with that, I would be happy to add ULID to Scramble.

I've tried to do that, but I didn't figure out a way to represent ULID in OpenAPI.

**Comment by maukoese** [comment]
Let me see what I can do. I'll try implement then see if I can make a PR.
